website/source/docs/platform/k8s/connect.html.md

```diff
@@ -206,6 +206,20 @@ Annotations can be used to configure the injection behavior.
         annotations:
           "consul.hashicorp.com/connect-service-upstreams":"[service-name]:[port]:[optional datacenter]"
         ```
+    * Consul Enterprise Namespaces
+    
+        If running Consul Enterprise 1.7+, your upstream services may be running in different
+        namespaces. The upstream namespace can be specified after the service name
+        as `[service-name].[namespace]`. See [Consul Enterprise Namespaces](#consul-enterprise-namespaces)
+        below for more details on configuring the injector.
```

Comment:
There needs to be a warning that adding a namespace to the annotation for older versions of Consul Enterprise won't work, nor will it work in OSS.

Reply:
Done.